### ISSUE TITLE
Weather Component

### ISSUE THREAD

**Issue by jinas123** [blade-ui-kit/blade-ui-kit#27]
A component to load weather from different providers.

**Comment by driesvints** [comment]
Yeah I can see a use case for that. Think we'll need to find a good and trustworthy API first.

**Comment by wsamoht** [comment]
I run a weather service for radio stations [Cirrus Forecast](https://cirrusforecast.com) built on Laravel and from my research the best free weather option is probably https://openweathermap.org. DarkSky was a good option before Apple bought it and discontinued the API. I have list of others too if you are interested.

**Comment by driesvints** [comment]
Thanks @wsamoht, that's helpful. Feel free to post some of the others.

**Comment by wsamoht** [comment]
Here is a full list that I have looked at that all have a free option.

Service|Free version # of API calls|Commercial Use
-|-|-
[OpenWeather](https://openweathermap.org/api)|1M/month|Yes
[Weatherbit](https://www.weatherbit.io/)|500/day|No (on free version)
[National Weather Service (USA)](https://www.weather.gov/documentation/services-web-api)|n/a|Yes
[climacell](https://www.climacell.co/weather-api/)|1,000/day|Yes
[Storm Glass](https://stormglass.io/)|50/day|No (on free version)
[weatherstack](https://weatherstack.com/)|1,000/month|No (on free version)

**Comment by jinas123** [comment]
@wsamoht  That's awesome!. I would personally recommend Open Weather
